docs/explore/base_image.rst

```diff
@@ -10,23 +10,78 @@ environment inside of each container. This guarantees you a truly blank slate;
 the dependencies you declare in your code directory are the only Python
 libraries that will be present. No more version clashes or surprises.
 
-Caliban uses one of two base images, depending on whether you're running in GPU
-(default) or CPU mode:
+Caliban uses a set of base images covering a set of common combinations of
+python and cuda versions. Our current matrix of supported combinations:
 
-* ``gcr.io/blueshift-playground/blueshift:gpu`` for the default GPU mode
-* ``gcr.io/blueshift-playground/blueshift:cpu`` for CPU, or ``--nogpu``\ , mode
++-----------+------------+------------+
```

Comment:
can you add a list of the images, or maybe link to the blueshift-playground folder where they live, so the user can figure it out themselves based on the matrix?

Reply:
good idea, I will add